kernel/config-5.11.x-x86_64

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.11.22 Kernel Configuration
+# Linux/x86 5.11.4 Kernel Configuration
```

Comment:
This does not look right.

Reply:
I was wondering about that too. The config currently has .22 but the Makefile mentions .4 https://github.com/linuxkit/linuxkit/blob/master/kernel/Makefile

Reply:
My understanding was that the previous configuration was incorrect

Reply:
Ah, looked more closely. We used to have a 5.11.22 kernel but I removed it with:
```
1e97e29be338 ("kernel: Remove 5.11.x as it is not longer maintained")
```
we still keep the config file around, though, since the real-time kernel (`-rt`) is still based on 5.11.x. For normal kernel we do check that the generated config matches the checked-in config to make sure we do not accidentally enable/disable kernel options, but for debug (`-dbg`) and real-time (`-rt`) kernels we don't because we append stuff to the config and expect to have difference.

Long story short, this looks l right